ethernet/w5500_spi/README.adoc

```diff
@@ -0,0 +1,88 @@
+= Attaching a W5500 via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to the W5500 chip. It is interfaced via SPI and power is supplied by the 3.3V pin.
+
+[NOTE]
+======
+This example simply sends and receives data from the W5500 chip using the polling approach. After you finish using the communication command* on the WIZ850io, ensure then the communication command set to clear.
+
+*The communication command is S"n"_CR (with "n" being the socket number).
+======
+
+== Importing external libraries
+
+The libraries required for this example are added to the .gitmodules file as a submodule and are imported automatically. 
+
+The structure used to generally add libraries to any project is:
+
+   The addition of 2 source libraries is demonstrated here. These lines can be added in any CMake file in the project. 
+   1) Add the source directory to the existing .gitmodules file through the following bash command.
+      $ git submodule add https://github.com/source/link
+
+   2) The source directory should be saved in the folder you are currently in. Set a path navigating to it. 
+      set(*source_directory_path /path/to/source/directory*)
+
+   3) Create a library.
+      add_library(*library_name* INTERFACE)
+
+   4) Decide which files you want to add to your library. 
+      target_sources(*library_name* INTERFACE *${source_directory_path}/source_file1.c ${source_directory_path}/source_file2.c*)
+
+   5) Add the source file directory to the library.
+      target_include_directories(*library_name ${source_directory_path}*)
+
+   Finally, remember to add the library_name to the target_link_libraries() command in the CMakeLists.txt file in the same directory as your target.
+   
+== Wiring information
+
+Wiring information for the WIZ850io board and the Raspberry Pi Pico Ethernet shield, both containing the W5500 chip, is provided below.
+
+WIZ850io board:
+More information is available [here](https://eshop.wiznet.io/shop/module/wiz850io/).
```

Comment:
Remember that this README.adoc will be imported into the PDF documentation. Please don't link using the word "here" or other methods like this, it's quite possible that this will be printed out, and in any case this is poor sentence construction.

Reply:
Right, should I be putting in link directly (no macro) or using a better descriptor than 'here'?